src/Aspire.Hosting.Azure.SignalR/AzureSignalRHealthCheck.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace Aspire.Hosting.Azure.SignalR;
+internal sealed class AzureSignalRHealthCheck : IHealthCheck
```

Comment:
Any thoughts on contributing this health check to https://github.com/xabaril/AspNetCore.Diagnostics.HealthChecks/? It doesn't need to block this PR, but the idea is that having a rich library of health checks (outside of Aspire) helps the whole community.

Reply:
Since Azure SignalR has it's own SDK with HealthCheck embeded, this class might be only useful for this emulator scenario.